stable/harbor/questions.yaml

```diff
@@ -0,0 +1,577 @@
+labels:
+  io.cattle.role: project # options are cluster/project
+questions:
+# Global Options
+- variable: harborAdminPassword
+  required: true
+  description: "The initial password of Harbor admin. Change it from portal after launching Harbor"
```

Comment:
I would say "Harbor admin user", but that's a nitpick

Reply:
done